Cabal-tests/tests/ParserTests/regressions/public-multilib-3.check

```diff
@@ -0,0 +1,3 @@
+No 'maintainer' field.
+No 'description' field.
+These packages miss upper bounds 'somelib' please add them using `cabal gen-bounds` for suggestions. For more information see:  https://pvp.haskell.org/
```

Comment:
Being a bit nit-picky, but this sentence lacks punctuation.
```
These packages miss upper bounds:
- somelib
Please add them, using ...
```
I am suggesting also to print the list via bullet points, makes it easier to read if there are many.
Yeah, why not make a test case (or modify this one) with a long list of packages that lack a bound?

Reply:
I changed this like you suggested

Reply:
@andreasabel are you happy with it now? Could you re-review again?